helm/cluster-azure/values.schema.json

```diff
@@ -273,70 +305,46 @@
                         "title": "Custom node taints",
                         "type": "array"
                     },
+                    "disableHealthCheck": {
+                        "title": "Disable HealthChecks for the MachineDeployment",
+                        "type": "boolean"
+                    },
                     "instanceType": {
                         "title": "VM size",
                         "type": "string"
                     },
                     "location": {
-                        "title": "Location",
+                        "title": "machinePool location",
                         "type": "string"
                     },
-                    "maxSize": {
-                        "title": "Maximum number of nodes",
-                        "type": "integer"
-                    },
-                    "minSize": {
-                        "title": "Minimum number of nodes",
-                        "type": "integer"
-                    },
                     "name": {
                         "description": "Unique identifier, cannot be changed after creation.",
                         "title": "Name",
                         "type": "string"
                     },
+                    "replicas": {
+                        "title": "Number of nodes in the MachineDeployment",
+                        "type": "integer"
+                    },
                     "rootVolumeSizeGB": {
                         "title": "Root volume size (GB)",
                         "type": "integer"
+                    },
+                    "type": {
```

Comment:
I think there should be a default value on the `type` property (probably `machineDeployment` for now).